include/lazy_importer.hpp

```diff
@@ -309,15 +309,15 @@ namespace li { namespace detail {
     LAZY_IMPORTER_FORCEINLINE const win::PEB_T* peb() noexcept
     {
 #ifdef _M_X64
-      return reinterpret_cast<const win::PEB_T*>(__readgsqword(0x60));
+        return reinterpret_cast<const win::PEB_T*>(__readgsqword(0x60));
 #elif _M_IX86
-      return reinterpret_cast<const win::PEB_T*>(__readfsdword(0x30));
+        return reinterpret_cast<const win::PEB_T*>(__readfsdword(0x30));
 #elif _M_ARM
-      return *reinterpret_cast<const win::PEB_T**>(_MoveFromCoprocessor(15, 0, 13, 0, 2) + 0x30);
+        return *reinterpret_cast<const win::PEB_T**>(_MoveFromCoprocessor(15, 0, 13, 0, 2) + 0x30);
 #elif _M_ARM64
-      return *reinterpret_cast<const win::PEB_T**>(__getReg(18) + 0x60);
+        return *reinterpret_cast<const win::PEB_T**>(__getReg(18) + 0x60);
 #elif _M_IA64
-      return *reinterpret_cast<const win::PEB_T**>((char*)_rdteb() + 0x60);
+        return *reinterpret_cast<const win::PEB_T**>((char*)_rdteb() + 0x60);
```

Comment:
C style cast >:|

Reply:
If intel docs are wrong and someone gets an error with my static_cast I'll be mildly unhappy